Jenkinsfile

```
@@ -4,7 +4,7 @@ pipeline {
         // Set the ssh key for the mirror using secret private key
         PRIVATE_KEY = credentials('EPITECH_SSH_KEY')
         PUBLIC_KEY = credentials('EPITECH_SSH_PUBKEY')
-        MIRROR_URL = 'git@github.com:EpitechPromo2027/B-FUN-400-NAN-4-1-compressor-axel.eckenberg.git'
+        MIRROR_URL = 'https://github.com/EpitechPromo2027/B-FUN-400-NAN-4-1-mypandoc-simon.ganier-lombard.git'
```

Please use this format, the mirror will not work else.
```suggestion
        MIRROR_URL = 'git@github.com:EpitechPromo2027/B-FUN-400-NAN-4-1-mypandoc-simon.ganier-lombard.git'
```